docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -79,6 +79,11 @@ Using a custom namespace solves problems later on because you do not have to ove
          # empty, disabled.
    ```
 
+   > **Note:** You can use `helm show values grafana/mimir-distributed` to see all configurable parameters for helm install.
+   > However, refrain from copy and paste the whole thing because all parameters default value are already set and you should only
+   > override the parameter that you want to change. Likewise, `values.yaml` from `operations/helm/charts/mimir-distributed/values.yaml`
+   > must be used only for matching helm chart version because it is not guaranteed to be backward compatible.
```

Comment:
```suggestion
   > override the parameter that you want to change.
   > If you want to get the values file from Github, build the URL as follows because we git tag every release:
   > `https://github.com/grafana/mimir/blob/mimir-distributed-<chart-version>/operations/helm/charts/mimir-distributed/values.yaml`.
   > For example, `https://github.com/grafana/mimir/blob/mimir-distributed-3.1.0/operations/helm/charts/mimir-distributed/values.yaml`. You must use a matching Helm chart version, because charts are not guaranteed to be backwards compatible.
```